charts/admission/charts/runtime/templates/deployment.yaml

```
@@ -20,6 +20,7 @@ spec:
         checksum/secret-kubeconfig: {{ include (print $.Template.BasePath "/secret-kubeconfig.yaml") . | sha256sum }}
         {{- end }}
       labels:
+        networking.gardener.cloud/to-runtime-apiserver: allowed
```

I think `networking.gardener.cloud/to-runtime-apiserver` is only needed for local setup where the kind cluster is also the virtual cluster.

You need 
```suggestion
        networking.resources.gardener.cloud/to-virtual-garden-kube-apiserver-tcp-443: allowed
```

in a setup with virtual cluster.

Can we add the `networking.gardener.cloud/to-runtime-apiserver: allowed` label somehow with skaffold? Because it should be only needed for local setup.